### DISCUSSION TITLE
Site not updating from GitHub trigger or manual

### DISCUSSION THREAD

**Discussion by mfreeman451** [mintlify/docs#110] [Bugs & Feedback]
As a few people have also pointed out in the Slack, Mintlify not getting updated by manual triggers or through the GitHub integration.

Please advise.

**Comment by sanchezfdezjavier** [comment]
Same issue here, we've just migrated our docs and we can't publish them 😢

**Comment by handotdev** [comment]
Hey everyone! Thanks for bringing this to our attention. Our team is currently investigating the source of the outage and have updated https://status.mintlify.com to reflect our updates

**Comment by devbrett90-prog** [comment]
minlify and venice ai are a hell of a cxombo
